### PR TITLE
feat: add mvn (Maven) completion spec

### PR DESCRIPTION
**Pull request by zp4rker**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
New completion spec

**What is the current behavior? (You can also link to an open issue here)**
Non-existent

**What is the new behavior (if this is a feature change)?**
Existent

**Additional info:**
Resolves https://github.com/withfig/autocomplete/issues/605

### CODE REVIEW

File: src/mvn.ts [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+        name: "server password",
+      },
+    },
+    {
+      name: ["--file", "-f"],
+      description:
+        "Force the use of an alternate POM file (or directory with pom.xml)",
+      args: {
+        name: "path",
+        suggestions: ["filepaths", "folders"],
```
Comment (QuiiBz) [inline src/mvn.ts:76]: This should actually be:

```suggestion
        template: ["filepaths", "folders"],
```

File: src/mvn.ts [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+      name: ["--show-version", "-V"],
+      description: "Display version information",
+    },
+    {
+      name: ["--debug", "-X"],
+      description: "Produce execution debug output",
+    },
+  ],
+  args: {
+    name: "goal/phase",
```
Comment (QuiiBz) [inline src/mvn.ts:220]: Could you also add a description for this argument?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this contribution!

File: src/mvn.ts (hunk 1 above)
Comment (zp4rker) [reply src/mvn.ts]: I blame GitHub Copilot 😂 

Fixed.

File: src/mvn.ts (hunk 2 above)
Comment (zp4rker) [reply src/mvn.ts:220]: Thanks for noticing, fixed!

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot for this contribution!

**Pull request merged**